CONTRIBUTING.md

```diff
@@ -112,7 +112,7 @@ exist to show coverage directly in your editor. E.g.:
 
 ### Documentation
 
-Here are some guidelines for writing documentation in Ratatui.  
+Here are some guidelines for writing documentation in Ratatui.
```

Comment:
I think we should prefer explicit new lines then rather than some quirky Markdown syntax. Markdown implementations differ a lot which also does not help with these smaller features.

New lines would end up creating a new paragraph so it should be either without new line or as a new paragraph in my opinion.

Reply:
I agree - here this line needs to be a new paragraph, but the lines below should be re-wrapped at 100 characters so the next line end up being at the end of the line where the double space was.

Reply:
Added suggestion https://github.com/ratatui-org/ratatui/pull/974/files/53f29f5b5fa74a10d47eebf36f66f2048bde7e71#r1509934442

Reply:
I applied the new wrapping in the way I think it was intended. Can you check if its correct?